src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -127,3 +127,19 @@
    :url          (get-in session [:peer :metadata :url])
    :accounts     (get-in session [:namespaces :eip155 :accounts])
    :disconnected false})
+
+(defn filter-operable-accounts
```

Comment:
Might be a good idea to keep this logic somewhere it can be reused. There's this function `status-im.contexts.communities.utils/sorted-operable-non-watch-only-accounts` that could reuse it for example. Maybe a good place to move it is in `wallet.data-store`?